### PR TITLE
Two small fixes: skip ci please

### PR DESCRIPTION
**Pull request by MattGal**
Remove EOL OSes from Archiver.csproj, update CoreCLR version to what's in the latest NetCore app package.

- If this PR should not run tests please add text "skip ci please" (remove the marked text, no quotes).
- Please add description for changes you are making.
- If there is an issue related to this PR, please add the reference.

### CODE REVIEW

**Comment by nguerrera** [comment]
@MattGertz FYI Infrastructure + approved dependency flow.

File: tools/Archiver/Archiver.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <OutputType>Exe</OutputType>
-    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;ubuntu.16.10-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.23-x64;fedora.24-x64;opensuse.13.2-x64;opensuse.42.1-x64;osx.10.10-x64;rhel.7-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.11-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;centos.7-x64;rhel.7.2-x64;debian.8-x64;fedora.24-x64;opensuse.42.1-x64;osx.10.10-x64;rhel.7-x64</RuntimeIdentifiers>
```
Comment (eerhardt) [inline tools/Archiver/Archiver.csproj]: Are these RIDs in other places as well?  I'm mostly thinking in tests.

**Comment by omajid** [comment]
Just a heads up. Fedora 24 has been EOL since 2017-08-08: https://fedoraproject.org/wiki/End_of_life. Also at https://fedoramagazine.org/fedora-24-eol/.

**Comment by MattGal** [comment]
@omajid good point, will address in a subsequent step (24 isn't removed from dependencies so it isn't blocking build atm).  Thanks!

**Pull request merged**